### ISSUE TITLE
keep older versions on the update site

### ISSUE THREAD

**Issue by carchrae** [eclipse-color-theme/eclipse-color-theme#152]
The update site contains only the latest version.  If a new version is broken, then there is no easy way to roll back.  (eg, see #149 )

**Comment by henriquedesousa** [comment]
Yes there are. It's crashing my RAD which is based on Eclipse 3.6, so I'd like to try the older versions.
